### PR TITLE
Add Product query, slug and usdPrice

### PR DESCRIPTION
**Pull request by jtaaa**
## What's changed

Added `product` query to product resolver and added `slug` and `usdPrice` to product model.
I also sneaked in a fix for `ImageCarousel` not displaying first image when images are added.

### CODE REVIEW

**Comment by jtaaa** [comment]
Task linked: [CU-b8n1y3 Add Product Form](https://app.clickup.com/t/b8n1y3)

File: packages/seller-client/src/generated/graphql.tsx [hunk 1]
```diff
@@ -243,8 +243,10 @@ export type NewProductInput = {
   name: Scalars['String'];
   onSale?: Maybe<Scalars['Boolean']>;
   price: Scalars['Float'];
+  slug: Scalars['String'];
```
Comment (chriskrogh) [inline packages/seller-client/src/generated/graphql.tsx:246]: what's a slug?

**Review by chriskrogh — APPROVED** [review]

File: packages/seller-client/src/generated/graphql.tsx (hunk 1 above)
Comment (jtaaa) [reply packages/seller-client/src/generated/graphql.tsx:246]: It's a the URL friendly version of the name that we will use in our marketplace routes to identify DB resources. A marketplace URL example for a product would be www.sellyourstuff.online/tifaraboutique/dream-cloud-trousers (the same as the beta)

Comment (jtaaa) [reply packages/seller-client/src/generated/graphql.tsx:246]: It's for SEO

**Pull request merged**